sdk/Internal/Clock.php

```diff
@@ -12,10 +12,10 @@ public function __construct()
     {
     }
 
-    public function millitime(): string
+    public function millitime(): int
     {
-        $millitime = microtime(true) * 1000;
+        $millitime = microtime(true) * 1e+6;
```

Comment:
Microseconds to milliseconds; should that be something like `microtime() / 1000`? There are 1000 microseconds to 1 millisecond, right?

Reply:
Yeah I was futzing with this for a couple hrtime implementation thoughts and never changed it back.  Will change it back and validate.

Reply:
Also I think you mean * 1000, as there are 1000 microseconds in a millisecond?